python_scripts/03_categorical_pipeline.py

```diff
@@ -276,30 +278,36 @@
 data["native-country"].value_counts()
 
 # %% [markdown]
-# We see that the `Holand-Netherlands` category is occurring rarely. This will
+# We see that the `"Holand-Netherlands"` category is occurring rarely. This will
 # be a problem during cross-validation: if the sample ends up in the test set
 # during splitting then the classifier would not have seen the category during
 # training and will not be able to encode it.
 #
-# In scikit-learn, there are two solutions to bypass this issue:
+# In scikit-learn, there are some possible solutions to bypass this issue:
 #
 # * list all the possible categories and provide it to the encoder via the
 #   keyword argument `categories`;
-# * use the parameter `handle_unknown`, i.e. if an unknown category is
+# * set the parameter `handle_unknown="infrequent_if_exist"`, i.e. if one or
+#   more unknown categories are encountered during transform, the resulting
+#   one-hot encoded columns will be mapped to the last position in the encoding.
+#   The notion of what is "infrequent" can be controlled with the
+#   hyper-parameter `min_frequency`.
```

Comment:
I think it would be easier to understand if we first explain what `min_frequency` does to then explain what `handle_unknown="infrequent_if_exist"` means.

```suggestion
# * it is possible to set `min_frequency` to a non-default value to
#   collapse the rarest categories observed on the training data
#   into a the last one-hot encoded feature. When this option is
#   enabled, it is then possible to also set
#   `handle_unknown="infrequent_if_exist"` to also encode the
#   unknown categories (categories only observed at predict time)
#   as ones in that last column.
```